File: unittest/scripts/py_dev_api_examples/results/Fetching_All_Data_Items_at_Once.py
myResult = myTable.select(['name', 'age']) \
  .where('name like :name').bind('name','S%') \
  .execute()
  
myRows = myResult.fetch_all()

for row in myRows:
  print("%s is %s years old." % (row.name, row.age))

File: unittest/scripts/js_dev_api_examples/results/Fetching_All_Data_Items_at_Once.js

var myResult = myTable.select(['name', 'age']).
  where('name like :name').bind('name','S%').
  execute();

var myRows = myResult.fetchAll();

for (index in myRows){
  print (myRows[index].name + " is " + myRows[index].age + " years old.");
}
